go/arrow/internal/flight_integration/scenario.go

```diff
@@ -2635,3 +2638,140 @@ func (m *flightSqlExtensionScenarioTester) ValidateTransactions(client *flightsq
 
 	return txn.Rollback(ctx)
 }
+
+type flightSqlIngestionScenarioTester struct {
+	flightsql.BaseServer
+}
+
+func (m *flightSqlIngestionScenarioTester) flightInfoForCommand(desc *flight.FlightDescriptor, schema *arrow.Schema) *flight.FlightInfo {
+	return &flight.FlightInfo{
+		Endpoint: []*flight.FlightEndpoint{
+			{Ticket: &flight.Ticket{Ticket: desc.Cmd}},
+		},
+		Schema:           flight.SerializeSchema(schema, memory.DefaultAllocator),
+		FlightDescriptor: desc,
+		TotalRecords:     -1,
+		TotalBytes:       -1,
+	}
+}
```

Comment:
this isn't actually being used anywhere, that's why the CI is failing

Reply:
Sorry, just removed.